typescript-style-backend.js

```diff
@@ -4,6 +4,19 @@ module.exports = {
             files: ['*.ts'],
             plugins: ['simple-import-sort'],
             rules: {
+                camelcase: [
+                    'error',
+                    {
+                        properties: 'always'
+                    }
+                ],
+                'no-underscore-dangle': [
```

Comment:
The error occurs during the call, not during the declaration.

![image](https://user-images.githubusercontent.com/18448635/189470390-3c72b8e4-2833-4870-a6fc-d75fe8e54910.png)